src/components/Report/index.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import './index.css';
 
 const Report = props => {
     return (
-        <div className="single-raport">
+        <div className={"single-raport " + (props.active ? 'active-report' : '')}>
```

Comment:
This is pure React & Redux way of styling components with conditional arguments. Good job. Also, if you would like to read more about it, please refer to this article:

https://medium.com/javascript-scene/10-tips-for-better-redux-architecture-69250425af44

and especially point number 5.

Also, using functional (stateless) component is better with Redux since it embraces better its paradigm.